lib/models/application.js

```diff
@@ -1650,6 +1661,131 @@ const getApplicationModel = function (deps, opts) {
 			remove: envVarModel.remove,
 		}),
 
+		/**
+		 * @namespace balena.models.application.buildEnvVar
```

Comment:
Not 100% sure about this, but for consistency I think the module's namespace should be `buildVar`.
What do you think?

Reply:
I thought to keep to the API model name *build*_*env*ironment_*var*iable. If you'd rather go with `buildVar` then I am not going to fight it 👍

Reply:
I'm fine with either :